src/components/Notification/NotificationDrawer/NotificationDrawerToggle.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { OverlayTrigger } from '../../OverlayTrigger';
+import { Tooltip } from '../../Tooltip';
+import { Icon } from '../../Icon';
+import { Button } from '../../Button';
+
+const NotificationDrawerToggle = ({ hasUnreadMessages, onClick }) => {
+  const iconName = hasUnreadMessages ? 'bell' : 'bell-o';
+  const tooltip = <Tooltip id="tooltip">Notifications</Tooltip>;
+
+  return (
+    <OverlayTrigger
+      placement="bottom"
+      id="notifications-toggle-icon"
+      overlay={tooltip}
+    >
+      <Button onClick={onClick} bsStyle="link">
+        <Icon
+          name={iconName}
+          style={{ color: 'rgb(240, 240, 240)', marginTop: '20px' }}
+          aria-describedby="tooltip"
+          size="lg"
```

Comment:
the `lg` should not be added, the mastheads determine the font size.

Reply:
Removing the `size` makes the bell icon too small, any recommendations?
I can make this component a "Storybook Only" component, by naming it and using it only there